types/src/build_version/mod.rs

```diff
@@ -17,13 +21,13 @@ const MINOR_ONLY_MASK: u32 = 0x00ff00;
 const PATCH_ONLY_MASK: u32 = 0x0000ff;
 
 // api versions
-const FULL_API_VERSION: Version = new_version(1, 1, 0);
-const MINER_API_VERSION: Version = new_version(0, 15, 0);
-const WORKER_API_VERSION: Version = new_version(0, 15, 0);
+const FULL_API_VERSION: Version = Version::new(1, 1, 0);
+const MINER_API_VERSION: Version = Version::new(0, 15, 0);
+const WORKER_API_VERSION: Version = Version::new(0, 15, 0);
```

Comment:
We should get rid of these, right?

Reply:
Not really, this indicates the versions of the RPC api:
In Lotus they are:
```
// semver versions of the rpc api exposed
var (
	FullAPIVersion0 = newVer(1, 5, 0)
	FullAPIVersion1 = newVer(2, 3, 0)

	MinerAPIVersion0  = newVer(1, 5, 0)
	WorkerAPIVersion0 = newVer(1, 6, 0)
)
```
In forest we can keep `1.1.0` for the full node rpc api. The other are less important since we don't have miner nor worker.

Reply:
I don't understand why we need them or what they mean. Are they required for interoperability with Lotus? Do we want interoperability with Lotus?